packages/story-editor/src/app/media/local/useContextValueProvider.js

```diff
@@ -261,7 +276,7 @@ export default function useContextValueProvider(reducerState, reducerActions) {
   // generate missing posters / has audio / base color if needed.
   useEffect(() => {
     media?.forEach((mediaElement) => postProcessingResource(mediaElement));
-  }, [media, mediaType, searchTerm, postProcessingResource]);
+  }, [media, mediaType, searchTerm, postProcessingResource, uploaded]);
```

Comment:
`uploaded` is not needed, as the media library doesn't upload on final upload, we need something to trigger reprocessing on final upload.

Reply:
The issue was, that now the media resources is not longer updated, with isTrimming etc, this effect was not firing when uploads were complete. State is not being updated as much, so this processing is not being trigger. Using this bit of state, that changes when an upload finishes, it trigger the backfill on upload finishes, which is what we expect to happen. If you remove this, then with it doesn't backfill when new files are uploaded, or backfills at the wrong time. Either way, it works fine.

Reply:
Okay, thanks for the explanation.

Follow-up question:if we implement #9739, backfilling for newly uploaded items would theoretically not be needed, right?

Reply:
> Follow-up question:if we implement #9739, backfilling for newly uploaded items would theoretically not be needed, right?

No, that isn't the base, images uploaded via wp media library always need backfill. The backfill is done to be sure that the many ways we upload assets, that that are always backfilled.

Reply:
Images uploaded via the WP media library change `uploaded` then?

Reply:
Media is also changed in that case. But we should look into this in a follow up ticket.

Reply:
Removed this now, it is not required